src/mqt/bench/benchmark_generator.py

```diff
@@ -328,10 +360,8 @@ def get_benchmark(  # noqa: PLR0911, PLR0912, PLR0915
         raise ValueError(msg)
 
     if compiler_settings is None:
-        compiler_settings = {
-            "qiskit": {"optimization_level": 1},
-            "tket": {"placement": "lineplacement"},
-        }
+        compiler_settings = CompilerSettings(QiskitSettings(1), TKETSettings("lineplacement"))
```

Comment:
Shouldn't these be default values for the respective settings? The data class definition seems like a more suitable place for this to me.
```suggestion
        compiler_settings = CompilerSettings(QiskitSettings(), TKETSettings())
```